client/my-sites/theme/themes-related-card/index.jsx

```diff
@@ -58,8 +62,8 @@ const ThemesRelatedCard = React.createClass( {
 					{ themes.map( theme => (
 						<li key={ theme.id }>
 							<Card className="themes-related-card__card">
-								<a href={ getDetailsUrl( theme ) }>
-									<img src={ theme.screenshot + '?w=' + '660' }/>
+								<a href={ this.props.getDetailsUrl( theme ) }>
+									<img src={ theme.screenshot + '?w=' + String( THEME_THUMBNAIL_WIDTH ) } />
```

Comment:
I don't think we need the `String` and I could go either way with using a constant here. It's up to you. I'd suggest going with 
```js
<img src={ `${ theme.screenshot }?w=660` />
or
<img src={ `${ theme.screenshot }?w=${ THEME_THUMBNAIL_WIDTH }` />
```